src/main/java/thePackmaster/patches/magnetizepack/OnDiscardPatch.java

```diff
@@ -0,0 +1,81 @@
+package thePackmaster.patches.magnetizepack;
+
+import basemod.abstracts.AbstractCardModifier;
+import basemod.helpers.CardModifierManager;
+import com.evacipated.cardcrawl.modthespire.Loader;
+import com.evacipated.cardcrawl.modthespire.ModInfo;
+import com.evacipated.cardcrawl.modthespire.lib.*;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.powers.AbstractPower;
+import javassist.*;
+import org.clapper.util.classutil.*;
+import thePackmaster.cardmodifiers.magnetizepack.MagnetizedModifier;
+import thePackmaster.util.Wiz;
+
+import java.io.File;
+import java.lang.reflect.Modifier;
+import java.net.URISyntaxException;
+import java.util.ArrayList;
+
+@SpirePatch(clz = CardCrawlGame.class, method = SpirePatch.CONSTRUCTOR)
```

Comment:
I can't use the same patch method as this other class because the uses of this patch need to be passed the card that was discarded to test for/trigger magnetized, while this other method doesn't have access to that reference.